.github/workflows/ci.yml

```diff
@@ -63,13 +102,13 @@ jobs:
     - uses: alehechka/upload-tartifact@v2
       if: always()
       with:
-        name: ${{ matrix.platform }}-JDK${{ matrix.jdk }}-${{ matrix.gradle_task }}-reports
+        name: windows-latest-JDK${{ matrix.jdk }}-${{ matrix.gradle_task }}-reports
         path: |
           ./build/reports/
 
   report-coverage:
     needs:
```

Comment:
Pretty sure we need `test-linux` too, otherwise it won't incorporate the run data from the linux runners

Reply:
> Pretty sure we need test-linux too, otherwise it won't incorporate the run data from the linux runners

Absolutely,  I am trying to understand why this job fails (no artifacts)